command/scripting.go

```diff
@@ -0,0 +1,370 @@
+package command
+
+import (
+	"context"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/evergreen-ci/evergreen/model"
+	"github.com/evergreen-ci/evergreen/rest/client"
+	"github.com/evergreen-ci/evergreen/util"
+	"github.com/google/shlex"
+	"github.com/mitchellh/mapstructure"
+	"github.com/mongodb/grip"
+	"github.com/mongodb/grip/level"
+	"github.com/mongodb/grip/message"
+	"github.com/mongodb/grip/send"
+	"github.com/mongodb/jasper/options"
+	"github.com/mongodb/jasper/scripting"
+	"github.com/pkg/errors"
+)
+
+type scriptingExec struct {
+	// Harness declares the implementation of the scripting
+	// harness to use to execute this code.
+	Harness string `mapstructure:"harness"`
```

Comment:
Since this is presumably a limited set of valid implementations, can we enumerate the allowed harnesses?